commit-branch.md

```diff
@@ -176,30 +176,36 @@ No tak to už umíme, tak si pojďme ten soubor commitnout příkazem:
 
 ```
 $ git add --all
-
-$ git commit
 ```
+
 **GOTCHA:** defaultní editor pro úpravu commit messages a vůbec všeho textového v Gitu je VIM.
-A nikdo neví, jak do něj něco napsat nebo hůř, jak se z něj dostat proto:
+A skoro nikdo neví, jak do něj něco napsat nebo hůř, jak se z něj dostat. Proto nastavíme náš oblíbený editor úpravou souboru ~/.gitconfig:
+```
+[core]
+  editor = nano
+```
```

Comment:
nechal bych tady údaje o tom, jak se odchází z VIMu, protože je to výchozí editor. Dá se předpokládá, že si ho tam většina lidí nechá, že se nebudou obtěžovat se změnou na `Nano`